plugins/modules/jenkins_node.py

```diff
@@ -75,6 +75,62 @@
         possible. In this case, a warning will be issued.
     type: str
     version_added: 10.0.0
+  launch_ssh:
+    description:
+      - When specified, sets the Jenkins node to launch by SSH.
+    type: dict
+    suboptions:
+      host:
+        description:
+          - When specified, sets the SSH host name.
+        type: str
+      port:
+        description:
+          - When specified, sets the SSH port number.
+        type: int
+      credentials_id:
+        description:
+          - When specified, sets the Jenkins credential used for SSH authentication by
+            its ID.
+        type: str
+      host_key_verify_none:
+        description:
+          - When set, sets the SSH host key non-verifying strategy.
+        type: bool
+        choices:
+          - True
```

Comment:
```suggestion
          - true
```
Is there a reason this can only be not set, or set to `true`?

Reply:
Good question.

Yes, all the ``host_key_verify_...`` options are mutually exclusive, and only some of them have sub-options i.e. the ones with no sub-options are effectively flags.